DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,25 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
-        if (!isset($config['default_formatter'])) {
-            reset($config['formatters']);
-            $config['default_formatter'] = key($config['formatters']);
-        }
-
         if (!array_key_exists($config['default_formatter'], $config['formatters'])) {
-            throw new \InvalidArgumentException(sprintf('SonataFormatterBundle - Invalid default formatter : %s, available : %s', $config['default_formatter'], json_encode(array_keys($config['formatters']))));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
Sometimes I wonder if consistency is that important: if you do something wrong and hard to fix many times, you just give up and carve in your standards : "do it the wrong way"